account_banking_sepa_credit_transfer/models/account_payment_order.py

```diff
@@ -11,6 +11,19 @@
 class AccountPaymentOrder(models.Model):
     _inherit = 'account.payment.order'
 
+    @api.model
+    def _get_line_key(self, line):
+        # The field line.date is the requested payment date
+        # taking into account the 'date_prefered' setting
+        # cf account_banking_payment_export/models/account_payment.py
+        # in the inherit of action_open()
+        return (
```

Comment:
Better to return a list, that it's dynamic

Reply:
The returned value is used as a ```dict``` key so it needs to be "hashable"

Reply:
A list is also hashable

Reply:
```
>>> {[1,2,3]: 'some value'}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'list'
```

Reply:
It's okay like that if the main purpose is to use as a key. You can always go from tuple to list and from list to tuple if you want to modify the values